inc/core/patterns/bcgov-alpha3-1x3-call-to-action.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Alpha V3 1x3 Call-to-Action.
+ *
+ * @since 1.7.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+ namespace Bcgov\Theme\Block;
+
+return [
+	'title'      => __( '1x3 Call-to-Action', 'bcgov_blocks_theme' ),
+	'categories' => [ 'bcgov-blocks-theme-general', 'bcgov-blocks-theme-alpha-v3' ],
```

Comment:
I like the idea of sharing the patterns between categories, but it may add load time to the pattern display/load time as each pattern gets rendered for the preview. Might be worth testing with users at some point.

Reply:
Yes, I'm up for testing this at some point. I do love redundant entries in different categories, I think it makes it easier to find things. Especially the alpha stuff, I think it should have a category. The general category might be less important, since we have the "All Patterns" category.